Filter fix (#20670)
demisto/content: Packs/MicrosoftGraphIdentityandAccess/ReleaseNotes/1_1_2.md

#### Integrations
##### Azure Active Directory Identity And Access

> - Fixed an issue with the fetch by adding a `$` to the filter.

Reviewer: ```suggestion
- Fixed an issue in the *fetch-incidents* where the filter did not work as expected.
```
